### DISCUSSION TITLE
Automatic switch from circular to up/down mode

### DISCUSSION THREAD

**Discussion by Mogli12** [Mogli12/AIVehicleExtension#287] [Announcements]
Build 14 contains a small experiment:

https://github.com/Mogli12/AIVehicleExtension/blob/master/Documents/circular_to_up_down.png

You can start in circular mode. AIVE will switch to up/down mode after the specified number of complete circles. Please make sure that the turn mode in the HUD for up/down is correct.

**Comment by headhunter84** [comment]
What a great feature! Thank you Mogli.
